### ISSUE TITLE
Liveness & Readiness support

### ISSUE THREAD

**Issue by kerthcet** [InftyAI/llmaz#21]
Add the support for inference services.

**Comment by pacoxu** [comment]
Also StartupProbe? See https://github.com/triton-inference-server/server/pull/5257/.

**Comment by kerthcet** [comment]
Yes, something like that, the core reason here is we should be aware of the server condition, ready or not? Maybe this can be part of the backendRuntime because it's related to the backend themselves.
